test/unit/data/data.spec.js

```diff
@@ -27,11 +27,11 @@ describe('Unit::Data', () => {
     it('creates a new Data instance', () => {
       const TestData = new Data(testApiKey, testSecret);
       expect(() => new Data(testApiKey, testSecret)).to.not.throw();
-      expect(TestData.metrics).to.be.an.instanceof(Metrics);
-      expect(TestData.errors).to.be.an.instanceof(Errors);
-      expect(TestData.exports).to.be.an.instanceof(Exports);
-      expect(TestData.videoViews).to.be.an.instanceof(VideoViews);
-      expect(TestData.filters).to.be.an.instanceof(Filters);
+      expect(TestData.Metrics).to.be.an.instanceof(Metrics);
+      expect(TestData.Errors).to.be.an.instanceof(Errors);
+      expect(TestData.Exports).to.be.an.instanceof(Exports);
+      expect(TestData.VideoViews).to.be.an.instanceof(VideoViews);
+      expect(TestData.Filters).to.be.an.instanceof(Filters);
```

Comment:
If we're keeping the lower-cased versions around for backwards compatibility, should we add tests to make sure those exist so we don't accidentally remove them for now?